src/systemcmds/tests/test_matrix.cpp

```diff
@@ -625,7 +625,7 @@ bool MatrixTest::vector2Tests()
 	ut_test(fabs(c(1) - 0) < 1e-5);
 
 	Matrix<float, 2, 1> d(a);
-	ut_test(fabs(d(0, 0) - 1) < 1e-5);
+	ut_test(fabs(d(0, 0) - 1.0f) < 1e-5);
```

Comment:
Ok I am confused because now it compiles with `ut_test(fabs(d(0, 0) - 1.0) < 1e-5);`, but not `ut_test(fabs(d(0, 0) - 1.0f) < 1e-5);`. So I must have made a mistake when uploading my code.

Either way this does not make sense to me, and it seems like an issue with the compiler trying to do optimizations that may reappear later.